### DISCUSSION TITLE
Jetbrains Extended Version of the Course

### DISCUSSION THREAD

**Discussion by gloc-mike** [talkpython/static-sites-with-sphinx-and-markdown-course#8] [General]
Hi @pauleveritt ,

I have continued on with the advanced (other parts) the course at Jetbrains (https://www.jetbrains.com/pycharm/guide/tutorials/sphinx_sites) and thought that it might not be appropriate to raise issues/ask questions regarding the other parts of the course that are not included in the Talk Python version.

Do you have a preferred method for raising issues/questions coming from the Jetbrains version of the course? Or are you okay for those questions to land here?

Thanks
Michael

**Comment by pauleveritt** [comment]
Whichever @mikeckennedy prefers. If you want less noise here, email me: first dot last at JetBrains dot com.

**Comment by mikeckennedy** [comment]
Hi guys. I think it's fine if you all keep the conversation going here as long as the thread makes it clear where it's coming from.
